packages/react-toolbar/Spec.md

```diff
@@ -2,62 +2,131 @@
+
+## Comparison of `@fluentui/react` and `@fluentui/react-northstar`
 
-- _Link to Open UI research_
-- _Link to comparison of v7 and v0_
-- _Link to GitHub epic issue for the converged component_
+- All mentions of v7 or v8 == `@fluentui/react` ([docsite](https://developer.microsoft.com/en-us/fluentui#/))
+- All mentions of v0 == `@fluentui/react-northstar` ([docsite](https://fluentsite.z22.web.core.windows.net/))
 
-## Sample Code
+The main difference between `@fluentui/react`'s `CommandBar` and `@fluentui/react-northstar`'s `Toolbar` is the right group of commands present in `CommandBar`.
 
-_Provide some representative example code that uses the proposed API for the component_
+v0 `Toolbar` has support to children API with static components in `Toolbar` as `Toolbar.Button` etc...
 
 ## Variants
 
-_Describe visual or functional variants of this control, if applicable. For example, a slider could have a 2D variant._
+The only layout variation is size differences, there are 2 sizes `medium` which is the default and `small`.
 
 ## API
 
-_List the **Props** and **Slots** proposed for the component. Ideally this would just be a link to the component's `.types.ts` file_
+The `Toolbar` will implement a `children` based API and will leverage the use of `context` in the interaction and data flows of child components.
 
-## Structure
+### Toolbar
 
-- _**Public**_
-- _**Internal**_
-- _**DOM** - how the component will be rendered as HTML elements_
+The root level component serves as a simple container for all possible ToolbarComponents
+
+```typescript
+type ToolbarProps = {
+  /**
+   * Defines toolbar size
+   * @default medium
+   */
+  size?: 'small' | 'medium';
+};
+```
+
+### ToolbarButton
+
+It serves as an override on top of `Button` limiting the possible props only to `size`, `appearance`, `disabled` and `disabledFocusable`
+
+```typescript
+type type ToolbarButtonProps = ComponentProps<ButtonSlots> &
+  Partial<Pick<ButtonProps, 'disabled' | 'disabledFocusable'>> & {
+
+  /**
+   * A button can have its content and borders styled for greater emphasis or to be subtle.
+   * - 'primary': Emphasizes the button as a primary action.
+   * - 'subtle': Minimizes emphasis to blend into the background until hovered or focused.
+   */
+  appearance?: 'primary' | 'subtle';
+;
+```
+
+### ToolbarDivider
+
+It's based on the `Divider` containing few styles overrides
+
+```typescript
+type ToolbarDividerProps = ComponentProps<Partial<DividerSlots>>;
+```
 
-## Migration
+### ToolbarRadioGroup
 
-_Describe what will need to be done to upgrade from the existing implementations:_
+It's based on the `RadioGroup` containing few styles overrides
+
+```typescript
+type ToolbarRadioGroupProps = ComponentProps<Partial<RadioGroupSlots>>;
```

Comment:
Shouldn't these just be:

```suggestion
type ToolbarRadioGroupProps = RadioGroupProps ;
```

Reply:
I think the `Partial` part here would make the difference, wouldn't?